homeassistant/components/lg_netcast/media_player.py

```diff
@@ -48,6 +48,7 @@
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_ACCESS_TOKEN): vol.All(cv.string, vol.Length(max=6)),
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_CAPTURE_SCREEN, default=True): cv.boolean,
```

Comment:
I think this falls under ADR 0010, and changes to YAML schema are no allowed.
If I am correct, you would need to migrate to config entries
https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

Reply:
@epenet is right. 
I plan to migrate the integration to config flow soon and we can make this option in the UI.

Reply:
@Drafteed if you would like and could give me a pointer in the right direction, I could look into migrating it, too.

Reply:
I would first plan to rewrite the backend to use fully asynchronous code, such like as https://github.com/home-assistant/core/pull/75727

Reply:
Started draft PR #78613.